src/lib/watch-processor.test.ts

```diff
@@ -218,24 +218,24 @@ describe("logEvent function", () => {
     const mockObj = { id: "123", type: "Pod" } as KubernetesObject;
     const message = "Test message";
     logEvent(WatchEvent.DATA, message, mockObj);
-    expect(Log.debug).toHaveBeenCalledWith(mockObj, `Watch event ${WatchEvent.DATA} received`, message);
+    expect(Log.debug).toHaveBeenCalledWith(mockObj, `Watch event ${WatchEvent.DATA} received. ${message}.`);
   });
 
   it("should handle CONNECT events", () => {
     logEvent(WatchEvent.CONNECT);
-    expect(Log.debug).toHaveBeenCalledWith(`Watch event ${WatchEvent.CONNECT} received`, "");
+    expect(Log.debug).toHaveBeenCalledWith(`Watch event ${WatchEvent.CONNECT} received.`);
```

Comment:
It seems kinda like this test isn't in sync with the recent changes since CONNECT events now propagate a url..?  Am I just looking at this wrong..?